### PR TITLE
Landing page merge ready

### PR DESCRIPTION
**Pull request by b0llu**
image
image

### CODE REVIEW

**Review by anik31 — APPROVED** [review]
Code is clean. Good work

File: ecom-css/footer.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  color: var(--white);
+  margin: 2rem 0;
+  font-weight: 400;
+}
+
+.footer-icon-container {
+  margin-left: auto;
+  margin-right: auto;
+  font-size: 3rem;
+  margin-bottom: 2rem;
```
Comment (partha8) [inline ecom-css/footer.css]: another way to write the margins is margin: 0 auto 2rem, just a suggestion as sudesh once mentioned about it.

**Review by partha8 — APPROVED** [review]
LGTM

File: ecom-css/footer.css (hunk 1 above)
Comment (b0llu) [reply ecom-css/footer.css]: yeah thanks for that partha this actually missed my eye

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me!

**Pull request merged**